site/content/docs/4.3/helpers/clearfix.md

```diff
@@ -2,7 +2,8 @@
 layout: docs
 title: Clearfix
 description: Quickly and easily clear floated content within a container by adding a clearfix utility.
-group: utilities
+group: helpers
+aliases: "/docs/4.3/helpers/"
```

Comment:
Do we really need the redirects since this will be a major version bump?

Reply:
I wasn't sure about this either, but since every css group redirects to it's first page, this would make sense.

Reply:
Ah, yeah, if it's for the first page, you should add it.